### PR TITLE
Spring cleanup of patches

### PR DESCRIPTION
**Pull request by dartasen**
Checked three times in a row :
- Make all class public (some were internal, some public, some without any identifier)
- Removed TARGET_CLASS for classthat doesn't need to keep a static variable
- Removed unused imports

Shouldn't we split this folder into several sub-folders (Cyclops, Prawn, Vehicles, BaseBuilding, ..) to improve readability?

### CODE REVIEW

File: NitroxPatcher/Patches/Dynamic/CyclopsShieldButton_OnClick_Patch.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@ public class CyclopsShieldButton_OnClick_Patch : NitroxPatch, IDynamicPatch
     {
         public static readonly Type TARGET_CLASS = typeof(CyclopsShieldButton);
```
[the remark this answers is not in the document]
Comment (dartasen) [reply NitroxPatcher/Patches/Dynamic/CyclopsShieldButton_OnClick_Patch.cs]: Done

**Comment by Jannify** [comment]
> Shouldn't we split this folder into several sub-folders (Cyclops, Prawn, Vehicles, BaseBuilding, ..) to improve readability?

I think this would be a great idea. `NitroxClient.GameLogic` files can probably give you an inspiration which categories to choose.
Keep in mind if you split the patches in sub-folders you should apply the same to `NitroxClient.Communication.Packets.Processors`, `NitroxServer.Communication.Packets.Processors`, `NitroxServer_Subnautica.Communication.Packets.Processors`, `NitroxModel.Packets`, `NitroxModel_Subnautica.Packets`.
But wait till somebody else approves this idea, don't want you to do work for nothing...

**Comment by Measurity** [comment]
When doing giant code file changes I think it's best to first discuss with everyone on what the preferred style is and should be. Changing access modifiers in 100+ files doesn't provide any benefit that one developer couldn't do in 1 second themself.

**Comment by dartasen** [comment]
Well we are here to discuss. If you're not happy with it just tell me to close it.

**Comment by Measurity** [comment]
Discussion becomes hard when you've already put a lot of work into something. It might seem that we judge you instead of the work. That's not the case.

However, I think it's best to close this PR and start with proper discussion before putting in the work so that you don't have to redo your work x times before we're all satisfied but instead just once, properly.

**Pull request closed without merging**